extensions/whatsapp/package.json

```diff
@@ -12,24 +12,17 @@
   "license": "MIT",
   "commands": [
     {
-      "name": "open-chat",
-      "title": "Open Chat",
-      "subtitle": "WhatsApp",
-      "description": "Open quickly added WhatsApp chats",
-      "mode": "view"
-    },
-    {
-      "name": "add-chat",
-      "title": "Add Chat",
-      "subtitle": "WhatsApp",
-      "description": "Add WhatsApp chats to open them later",
+      "name": "open-in-whatsapp",
```

Comment:
Can we keep this as it was before? `Open Chat` and `WhatsApp` as subtitle. This matches how most extension commands are defined.